manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -19,6 +19,26 @@ spec:
       app: istiod
       istio: pilot
 ---
+# Source: istio-discovery/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-reader-servive-account
```

Comment:
do we really want to revision the service account? Aside from backwards compat issues with multicluster, it also changes the identity. I guess that matters less since istiod doesn't have a sidecar..

The alternative I think is to have a single service account, and it gets the superset of allowed permissions